.rspec

```diff
@@ -1,6 +1,3 @@
---format documentation
```

Comment:
I think it's nice to have documentation format, but it's your call

Reply:
I don't really like it for normal test runs since it makes it impossible to see where the progress is. You can always pass it if you want to see the doc format too.